rest/data/project_test.go

```diff
@@ -245,8 +225,8 @@ func (s *ProjectConnectorGetSuite) TestGetProjectEvents() {
 		s.Equal(evergreen.RedactedAfterValue, eventLog.After.Vars.Vars["hello"])
 		s.Equal(evergreen.RedactedBeforeValue, eventLog.Before.Vars.Vars["world"])
 		s.Equal(evergreen.RedactedAfterValue, eventLog.After.Vars.Vars["world"])
-		s.Equal(utility.FromStringPtr(eventLog.Before.GithubAppAuth.PrivateKey), "")
-		s.Equal(utility.FromStringPtr(eventLog.After.GithubAppAuth.PrivateKey), evergreen.RedactedValue)
+		s.Empty(utility.FromStringPtr(eventLog.Before.GithubAppAuth.PrivateKey))
+		s.Equal(evergreen.RedactedAfterValue, utility.FromStringPtr(eventLog.After.GithubAppAuth.PrivateKey))
```

Comment:
can we add a test for the before value as well?

Reply:
That's already tested on L228.